Source/items.cpp

```diff
@@ -2065,12 +2098,13 @@ static void SaveItemSuffix(int i, int sufidx)
 
 	SaveItemPower(
 	    i,
-	    PL_Suffix[sufidx].PLPower,
+        {PL_Suffix[sufidx].PLPower,
 	    param1,
 	    param2,
 	    PL_Suffix[sufidx].PLMinVal,
 	    PL_Suffix[sufidx].PLMaxVal,
-	    PL_Suffix[sufidx].PLMultVal);
+	    PL_Suffix[sufidx].PLMultVal});
+	// TODO : Function to make params

```

Comment:
This could be an option:
```cpp
auto params = GetPrefixPowerParams(PL_Suffix[sufidx]);
params.randMin = param1;
params.randMax = param2;
SaveItemPower(i, params);
```

Reply:
Applied, changed the name of the function to "SaveItemPowerParams::fromXfixItem(...)" as it's for box prefix and suffix, not sure if we could name it based on "PLStruct", it would be helpful if I knew what that stands for 🤣